readme.md

```diff
@@ -137,7 +137,7 @@ Click the type names for complete docs.
 
 ### Related
 
-- [typed-query-selector](https://github.com/g-plane/typed-query-selector) - Enhances `querySelector` and `querySelectorAll` with a template literal type that matches element types returned from an HTML element query selector.
+- [typed-query-selector](https://github.com/g-plane/typed-query-selector) - Enhances `document.querySelector` and `document.querySelectorAll` with a template literal type that matches element types returned from a HTML element query selector.
```

Comment:
I think *[...] an HTML [...]* is the correct form...

https://english.stackexchange.com/questions/1016/do-you-use-a-or-an-before-acronyms-initialisms